### ISSUE TITLE
other conversions to be default, some other ideas

### ISSUE THREAD

**Issue by jirafey** [mozeeshan1/VxT-Python#1]
Domains to convert:
- twitter.com : vxtwitter.com
- x.com : vxtwitter.com
- instagram.com : ddinstagram.com
- tiktok.com : tiktxk.com
- reddit.com : rxddit.com

**Comment by jirafey** [comment]
also the main thing is that you cant really reply to original messenger with ping just by @ is there a way to implement it?

wanted to make a first issue here, this project is great :D

**Comment by mozeeshan1** [comment]
That's a great suggestion! Thank you for the suggestion and ur kind words. I'll add Reddit conversion to it for the next update but the rest will stay the same. I'll look into implementing an auto ping if you reply to the bot for the next update as well.

**Comment by jirafey** [comment]
my idea would be just appending @user to the message the other way around i have no idea how but ll be happy if you succeed. Merry Christmas!

**Comment by mozeeshan1** [comment]
Yeah, that'd be a simpler way to do it, but I'll look into doing it with the message properties when the bot sends it. 

Hope you have a good holiday season!

**Comment by jirafey** [comment]
> Yeah, that'd be a simpler way to do it, but I'll look into doing it with the message properties when the bot sends it.
> 
> Hope you have a good holiday season!

hey how's it going, was there any work been done in this issue's direction?

**Comment by mozeeshan1** [comment]
Hey, thank you for being so patient and checking up on the status of this issue! Sorry about the delay. I'm caught up in many other projects at the moment, but I plan on adding this to the features once I update the bot to a newer version. Expect it to be out by the end of February. It's pretty big changes on the infrastructure side so it'll take a while as well.

**Comment by jirafey** [comment]
> Hey, thank you for being so patient and checking up on the status of this issue! Sorry about the delay. I'm caught up in many other projects at the moment, but I plan on adding this to the features once I update the bot to a newer version. Expect it to be out by the end of February. It's pretty big changes on the infrastructure side so it'll take a while as well.

ok, no worries :3
